language/stdlib/modules/LibraSystem.move

```diff
@@ -13,34 +15,40 @@ module LibraSystem {
     use 0x1::Roles;
     use 0x1::LibraTimestamp;
 
-    /// ValidatorInfo contains information about a Validator Owner.
+    /// Information about a Validator Owner.
     struct ValidatorInfo {
         /// The address (account) of the Validator Owner
         addr: address,
         /// The voting power of the Validator Owner (currently always 1).
         consensus_voting_power: u64,
-        /// Configuration information about the Validator.
+        /// Configuration information about the Validator, such as the
+        /// Validator Operator, human name, and info such as consensus key
+        /// and network addresses.
+        ///
+        /// > Note: `ValidatorConfig` is different from `LibraConfig`.
```

Comment:
I don't understand this note. Obviously they are different because already the type names are different.